storage/migrations/02_oasis_3_emerald.up.sql

```diff
@@ -2,8 +2,11 @@
 
 BEGIN;
 
-CREATE TABLE oasis_3.emerald_rounds
+CREATE TYPE runtime AS ENUM ('emerald', 'sapphire', 'cipher');
+
+CREATE TABLE oasis_3.runtime_blocks
```

Comment:
what's the rounds -> blocks change? seems fine btw

Reply:
Not strictly related to the rest of this PR. It's just for consistency; most of our internal code and the API all refer to this as blocks.